runtime/doc/if_lua.txt

```diff
@@ -293,7 +353,20 @@ vim.types						*lua-vim.types*
 	values corresponding to `vim.types.float`, `vim.types.array` and 
 	`vim.types.dictionary` will not change or that `vim.types` table will 
 	only contain values for these three types.
+------------------------------------------------------------------------------
+vim.helpers.* helper functions
+
+This modules provides lua-only helper functions to assist in writing lua
+plugins.
+
+vim.helpers.new_plugin({string})                        *vim.helpers.new_plugin*
```

Comment:
This would live on `vim.new_plugin`, we would not have a "helpers" module.

Reply:
Right, I did not bother to move things around properly, it's mainly the idea that I wanted to keep around :)